tabs/auxiliary.js

```diff
@@ -36,28 +36,16 @@ TABS.auxiliary.initialize = function (callback) {
         return false;
     }
 
-    function adjustRunCamSplitBoxNameWithModeID(modeId, originalModeName) {
-        switch (modeId) {
-            case 32: // BOXCAMERA1
-                return "CAMERA WI-FI";
-            case 33: // BOXCAMERA2
-                return "CAMERA POWER";
-            case 34: // BOXCAMERA3
-                return "CAMERA CHANGE MODE"
-            default:
-                return originalModeName;
-        }
+    function localizeBoxNameWithModeID(modeId, originalModeName) {
```

Comment:
This is wrong. These switches were defined for generic use for cameras in the firmware for a reason. If this behaviour is not wanted any more, the switches should be renamed to be specific to RunCam in the firmware.

I think the original intent to only override the labels if a port for a RunCam is configured is correct, and if the labels do not get propery updated in this case this (and only this) should be fixed.

Reply:
Aha, I didn't knew that these camera switches where intended to be used for generic camera controls, I thought there were specific to the RunCam Split. That figures why the code was setup like it was. I will revert this back and fix the update issue, but I will leave in the localized part (have to change it a little bit).

The update issue is probably the case that the ports config is not yet loaded from the fc. Port config is only loaded from the fc in specific tabs, that is why that after opening another tab, the labels are displayed correctly.

Reply:
Yes, adding the messages to localisation is definitely a good and wanted change. And I think you are right, the port configuration not having been loaded is most likely the culprit. There normally is a string of MSP commands in the js for every tab that are sent whenever the tab is opened, and it should be sufficient to copy / paste the one MSP command that loads the ports config from the ports tab into the modes tab.